test/e2e/snaps/test-snap-uilinks.spec.js

```diff
@@ -0,0 +1,129 @@
+const {
+  defaultGanacheOptions,
+  withFixtures,
+  unlockWallet,
+  WINDOW_TITLES,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const { TEST_SNAPS_WEBSITE_URL } = require('./enums');
+
+describe('Test Snap Dialog', function () {
+  it('test all three snap_dialog types', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        failOnConsoleError: false,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+
+        // navigate to test snaps page and connect to dialog snap
```

Comment:
Does this need to be a separate test?

Reply:
Our original discussion on the topic landed on creating a links test seperately, future-proofing for other possible cases where links could happen, making seperate  ITs if necessary, but this one, at this point, only checks for a link in one of the three dialog types (where it has already been added to the example snap.)

Reply:
It also leaves this test open to testing other UI features if necessary